internal/stackdriver/stackdriver.go

```diff
@@ -44,6 +44,46 @@ func NewAPIClient(ctx context.Context, project string) (*API, error) {
 	}, nil
 }
 
+// RequestCount count returns the number of requests for the given offset and
+// query.
+func (a *API) RequestCount(ctx context.Context, query Query, offset time.Duration) (int64, error) {
+	query = addFilterToQuery(query, "metric.type", requestCount)
+	endTime := time.Now()
+	startTime := endTime.Add(-1 * offset)
+
+	it := a.MetricClient.ListTimeSeries(ctx, &monitoringpb.ListTimeSeriesRequest{
```

Comment:
use this as it doesn't return an iterator
https://godoc.org/google.golang.org/api/monitoring/v3#ProjectsTimeSeriesListCall.Do

Reply:
My bad. I just noticed that this package is deprecated. That's why it looked a little bit weird to me yesterday. We are using [this one](https://pkg.go.dev/cloud.google.com/go/monitoring/apiv3?tab=doc) as recommended.

Reply:
If you think new pkg has all the capabilities use it. I suspect it's a handwritten pkg and we tend to have potential bugs or future deprecations as you've seen :)